corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -116,6 +116,7 @@ def __call__(self, fn):
 INSTANCE_BY_ID = {
     'groups': Instance(id='groups', src='jr://fixture/user-groups'),
     'reports': Instance(id='reports', src='jr://fixture/commcare:reports'),
+    'locations': Instance(id='locations', src='jr://fixture/commtrack:locations'),
```

Comment:
if we are adding this can we also add support for the [revised location fixture format](https://github.com/dimagi/commcare-hq/pull/12543)?

confusingly that is `jr://fixture/locations` - i don't actually know how the ID side of this dict is used, but we might want to design some kind of plan for that now before getting ourselves into trouble as I think the long term plan is to deprecate `commtrack:locations` and move to the flat format